src/test/resources/features/refapp-3.x/06-patient-search/patient-search.feature

```diff
@@ -3,10 +3,7 @@ Feature: Patient Search
   Background:
     Given the user login to the Registration Desk
 
-  Scenario Outline: Search for a patient
-    When the user search for "<patientName>"
-    Then the result should be "<result>"
-    Examples:
```

Comment:
isnt the user supposed to enter patient to be searched here

Reply:
It was, from this PR the patient name is randomly generated, and we use that patient name throughout the rest of the test and delete the patient at the end of the test.

Reply:
Sounds better.thanks